software/webapp/src/main/java/brooklyn/entity/webapp/nodejs/NodeJsWebAppSshDriver.java

```diff
@@ -152,6 +152,8 @@ public void launch() {
             appCommandLine = appCommand + " " + appFile;
         }
 
+        // Ensure global NPM modules are on Node's path.
+        commands.add("export NODE_PATH=\"$NODE_PATH:$(npm root -g)\"");
```

Comment:
I think we could rework the Node entity to install packages locally. Something for the future.